packages/ubuntu_desktop_installer/lib/pages/installation_slides/installation_slides_page.dart

```diff
@@ -160,16 +179,12 @@ class _ExpandButton extends StatelessWidget {
         child: Row(
           mainAxisSize: MainAxisSize.min,
           children: <Widget>[
-            SizedBox(
-              width: _kExpandIconSize,
-              height: _kExpandIconSize,
-              child: ExpandIcon(
-                color: Colors.white,
-                size: _kExpandIconSize,
-                padding: EdgeInsets.zero,
-                isExpanded: isExpanded,
-                onPressed: (_) => onExpanded(!isExpanded),
-              ),
+            AnimatedContainer(
+              curve: Curves.fastOutSlowIn,
+              duration: kThemeAnimationDuration,
+              child: Icon(Icons.arrow_right),
```

Comment:
`ExpandIcon` was replaced by a plain rotating icon because `ExpandIcon` has a massive splash radius and the icon cannot be changed...
![image](https://user-images.githubusercontent.com/140617/161306432-be3c807b-b7c5-45f8-9662-96204bd01590.png)